service/templates/admin.html

```diff
@@ -350,17 +361,6 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 													</md-radio-group>
 												</td>
 											</tr>
-											<tr>
-												<td>Projects Track Multiple Branches</td>
```

Comment:
Why are you removing this?

Reply:
Sorry this is unrelated, just dead code I noticed while testing; should've been removed earlier.